packages/core/src/html/includePanelProcessor.ts

```diff
@@ -230,6 +230,24 @@ export function processInclude(node: MbNode, context: Context, pageSources: Page
     const $ = cheerio.load(actualContent);
     const actualContentOrNull = $(hash).html();
     actualContent = actualContentOrNull || '';
+    if (actualContent !== '') {
+      const footnotePattern = /<a aria-describedby="footnote-label" href="#fn-\d+-\d+">\[\d+\]/g;
+      const matchs = actualContent.match(footnotePattern);
+      if (matchs) {
```

Comment:
nit: the plural of match is matches. can you change the instances of `matchs` to `matches`?